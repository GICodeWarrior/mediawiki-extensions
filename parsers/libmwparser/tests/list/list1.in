hello
* list
** list item
* bar
*# baz
*# quiz
  :::: { foo
:;;; bar
