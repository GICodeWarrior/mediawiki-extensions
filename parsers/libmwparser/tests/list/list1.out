BEGIN ARTICLE
    BEGIN PARAGRAPH
        WORD[hello]
    END PARAGRAPH
    BEGIN BULLET LIST
        BEGIN BULLET LIST ITEM
            WORD[list]
        END BULLET LIST ITEM
    END BULLET LIST
    BEGIN BULLET LIST
        BEGIN BULLET LIST ITEM
            BEGIN BULLET LIST
                BEGIN BULLET LIST ITEM
                    WORD[list]
                    SPACE[ ]
                    WORD[item]
                END BULLET LIST ITEM
            END BULLET LIST
        END BULLET LIST ITEM
    END BULLET LIST
    BEGIN BULLET LIST
        BEGIN BULLET LIST ITEM
            WORD[bar]
        END BULLET LIST ITEM
    END BULLET LIST
    BEGIN BULLET LIST
        BEGIN BULLET LIST ITEM
            BEGIN ENUMERATION LIST
                BEGIN ENUMERATION LIST ITEM
                    WORD[baz]
                END ENUMERATION LIST ITEM
            END ENUMERATION LIST
        END BULLET LIST ITEM
    END BULLET LIST
    BEGIN BULLET LIST
        BEGIN BULLET LIST ITEM
            BEGIN ENUMERATION LIST
                BEGIN ENUMERATION LIST ITEM
                    WORD[quiz]
                END ENUMERATION LIST ITEM
            END ENUMERATION LIST
        END BULLET LIST ITEM
    END BULLET LIST
    BEGIN PARAGRAPH
        SPECIAL[::::]
        SPACE[ ]
        SPECIAL[{]
        SPACE[ ]
        WORD[foo]
    END PARAGRAPH
    BEGIN DEFINITION LIST
        BEGIN DEFINITION ITEM
            BEGIN DEFINITION LIST
                BEGIN DEFINITION LIST
                    BEGIN DEFINITION LIST
                        BEGIN DEFINED TERM ITEM
                            WORD[bar]
                        END DEFINED TERM ITEM
                    END DEFINITION LIST
                END DEFINITION LIST
            END DEFINITION LIST
        END DEFINITION ITEM
    END DEFINITION LIST
END ARTICLE
